<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>751</width>
    <height>674</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>641</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>1. Background Substraction</string>
    </property>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>621</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string> 1.1 Background Substraction</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>140</y>
      <width>641</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string>2. Optical flow</string>
    </property>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>621</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>2.1 Preprocessing</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>621</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>2.2 Video tracking</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>330</y>
      <width>641</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>3. Perspective Transform</string>
    </property>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>621</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>3.1 Perspective Transform</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>460</y>
      <width>641</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string>4. PCA</string>
    </property>
    <widget class="QPushButton" name="pushButton_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>621</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>4.1 Image Reconstruction</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>621</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>4.2 Compute the Reconstruction Error</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>751</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
